roles/wildfly_systemd/tasks/yml_config.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Enable extension in wildfly
   when: (wildfly_version.split('.')[:3] | join('.') is version('26.0.1', '>=')) or (wildfly_version.split('.')[:1] | join('.') is version('8', '>='))
-  become: yes
+  become: true
```

Comment:
Same as usual, I would like to add a var so user can set become: to no if their env does not require (and more crucially does not give them) root priv.